profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -738,6 +750,121 @@ function tmgmt_poetry_page_alter(&$page) {
 /**
  * Implements hook_form_FORM_ID_alter().
  *
+ * Change the #cart form to provide the tmgmt_poetry module functionalities.
+ */
+function tmgmt_poetry_form_tmgmt_ui_cart_content_alter(&$form, &$form_state) {
+  $form['request_translation']['#submit'] = ['_tmgmt_poetry_workbench_store_request_languages_callback'];
+}
+
+/**
+ * Change the #cart submit logic to provide tmgmt_poetry module functionalities.
+ *
+ * @see tmgmt_ui_cart_request_translation_form_submit()
+ */
+function _tmgmt_poetry_workbench_store_request_languages_callback(&$form, &$form_state) {
+  global $user;
+
+  $target_languages = array_filter($form_state['values']['target_language']);
+  $enforced_source_language = NULL;
+  if ($form_state['values']['enforced_source_language']) {
+    $enforced_source_language = $form_state['values']['source_language'];
+  }
+
+  $skipped_count = 0;
+  $job_items_by_source_language = array();
+  // Group the selected items by source language.
+  foreach (tmgmt_job_item_load_multiple(array_filter($form_state['values']['items'])) as $job_item) {
+    $source_language = $enforced_source_language ? $enforced_source_language : $job_item->getSourceLangCode();
+    if (in_array($source_language, $job_item->getExistingLangCodes())) {
+      $job_items_by_source_language[$source_language][$job_item->tjiid] = $job_item;
+    }
+    else {
+      $skipped_count++;
+    }
+  }
+
+  $jobs = array();
+  $remove_job_item_ids = array();
+  // Loop over all target languages, create a job for each source and target
+  // language combination add add the relevant job items to it.
+  foreach ($target_languages as $target_language) {
+    foreach ($job_items_by_source_language as $source_language => $job_items) {
+      // Skip in case the source language is the same as the target language.
+      if ($source_language == $target_language) {
+        continue;
+      }
+
+      if (isset($reference)) {
```

Comment:
Could you show me where in the scope of this function you are assigning a value to it?